### ISSUE TITLE
Italian installer issue

### ISSUE THREAD

**Issue by samo79** [jens-maus/libopenurl#11]
The installer work and install everything correctly, but it doesn't seems to pick the italian strings.. aka all the installing procedure need to be done in english

**Comment by samo79** [comment]
Noted also that by using the installer, when we choose to install the Reaction GUI instead of the MUI one, then no icon will be copied in SYS:Prefs (only the exe will be copied there)

**Comment by tboeckel** [comment]
Well, the Installer script simply copies all files from the source directory to the destination directory. For AmigaOS4 the source directory is either Prefs/AmigaOS4/MUI or Prefs/AmigaOS4/ReAction and according to the archive contents an icon is contained in both.
The only possible explanation could be the use of the copylib command to copy the files. This might skip files which do not contain any version information, like icons. copyfiles could be used instead. But this does not explain why this should make a difference between directories, because the icon file is the same in all places.

**Comment by samo79** [comment]
And what about the problem with the localization of the installer ?
I did a quick test with the other installer i've translated for you, the one for the MUI classes, the main MUI package, Flexcat etc.. all fine

Mmm btw.. tried to reinstall OpenURL from scratch, choosed Reaction and continue tiil the end.. and can confirm that the icon will not be copied in Prefs directory.. instead it will be copied ok if i choose to install the MUI GUI

**Comment by samo79** [comment]
Mmm look at the installer code and maybe found the issue :

(if (= #language "italiano")

Language name should be "italian", not "italiano"

**Comment by samo79** [comment]
Just tried to set my system in german, then i run the installer.. the installer seems to pick german language correctly .. so i've checked if maybe i did some mistake on the italian part but all seems correct ... at this point there should be an error elsewhere

**Comment by tboeckel** [comment]
Have you tried to swap the checks for italian and german but keeping the translations in place? Does that make any difference? Sorry, but I am really lacking the time to check this myself.

**Comment by samo79** [comment]
Yes tried also that.. changed the order, also tried to change the detection numbers, copy the entire german text over the italian etc.. no luck, everything still in english and it works also in german if i change my system to use this language.. but italian not
All the other installers you did works perfectly in italian, this one for some reason no
